### PR TITLE
Add pontoon script

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@ScottDowne and @jbuck R?

File: components/pontoon.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+var production = process.env.NPM_CONFIG_PRODUCTION === 'false' ? false : true;
+
+var Pontoon = React.createClass({
+  render: function() {
+    if (!production) {
+      return (
+        <script src='https://pontoon.mozilla.org/pontoon.js'></script>
```
Comment (ScottDowne) [inline components/pontoon.jsx]: Why do we need this on staging and local dev but not production?

File: pages/index.jsx [hunk 2]
```diff
@@ -17,6 +18,7 @@ var Index = React.createClass({
           <script src="https://js.stripe.com/v1/"></script>
           <script src="/js/stripe.js"></script>
           <script src="/js/ga.js"></script>
+          <Pontoon/>
```
Comment (ScottDowne) [inline pages/index.jsx]: What's the downside to just adding `<script src='https://pontoon.mozilla.org/pontoon.js'></script>` always?

File: components/pontoon.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+var production = process.env.NPM_CONFIG_PRODUCTION === 'false' ? false : true;
```
Comment (jbuck) [inline components/pontoon.jsx]: Change the name of this to `ENABLE_PONTOON` or something along those lines

**Comment by alicoding** [comment]
@jbuck can you R? again?

**Comment by jbuck** [comment]
@alicoding r+

File: components/pontoon.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+var production = process.env.ENABLE_PONTOON === 'false' ? false : true;
```
Comment (jbuck) [inline components/pontoon.jsx]: `=== 'true'`

**Pull request merged**